mis_builder/models/aep.py

```diff
@@ -543,4 +543,4 @@ def get_unallocated_pl(cls, companies, date):
         # TODO shoud we include here the accounts of type "unaffected"
         # or leave that to the caller?
         bals = cls._get_balances(cls.MODE_UNALLOCATED, companies, date, date)
-        return tuple(map(sum, zip(*bals.values(), strict=True)))
+        return tuple(map(sum, zip(*bals.values())))  # noqa
```

Comment:
```suggestion
        return tuple(map(sum, zip(*bals.values())))  # noqa: XXXX
```

Silence the specific warning, for documentation and precision.